Student_BCOs/Derek_Cao_BCO 000534.json

```diff
@@ -0,0 +1,175 @@
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "10/16/2019"
+            }
+        ],
+        "contributors": [
+            {
+                "name": "Derek Cao",
+                "orcid": "https://orcid.org/0000-0002-7901-9470",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "cderek7@gwu.edu"
+            },
+            {
+                "name": "Tiangong Lu",
+                "orcid": "",
+                "affiliation": "Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor",
+                "contribution": [
+                    "authoredBy",
+                    "derivedFrom"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Armand Bankhead, III,",
+                "orcid": "",
+                "affiliation": "Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor ; Department of Biostatistics, School of Public Health, University of Michigan, Ann Arbor",
+                "contribution": [
+                    "authoredBy",
+                    "derivedFrom"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Mats Ljungman ",
+                "orcid": "https://orcid.org/0000-0003-1553-6695",
+                "affiliation": "Departments of Radiation Oncology, Rogel Cancer Center, University of Michigan Medical School and Department of Environmental Health Sciences, School of Public Health, University of Michigan, Ann Arbor",
+                "contribution": [
+                    "authoredBy",
+                    "derivedFrom"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Nouri Neamati",
+                "orcid": "https://orcid.org/0000-0003-3291-7131",
+                "affiliation": "Department of Medicinal Chemistry, College of Pharmacy, Rogel Cancer Center, University of Michigan, Ann Arbor",
+                "contribution": [
+                    "authoredBy",
+                    "derivedFrom"
+                ],
+                "email": ""
+            }
+        ]
+    },
+    "usability_domain": [
+        "Adapted from Tiangong Lu, Armand Bankhead, III,  Mats Ljungman, Nouri Neamati. \"Multi-omics profiling reveals key signaling pathways in ovarian cancer controlled by STAT3.\" PMC6735387(July 28, 2019). https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6735387/",
+        "Identify multi expression level of Bru-Seq, RNA-Seq, and MS Proteomic. ",
```

Comment:
Thank you for modifying the usability domain with your own words. However, some of this just doesn't make sense to me. For example: Bru-Seq is a technique. So I'm not sure how you can identify multi expression level of Bru-Seq.

I have gone through several review cycles for this BCO, and we don't have time for one more before the pFDA challenge deadline. Given I cannot give my approval for this BCO in its current state, I will no longer be a reviewer on this BCO. **My name should not be listed as a reviewer on the BCO you submit to the pFDA challenge**


Please remove the following content from the "review" section: 
- "status": "in-review"
"reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
                    "orcid": "https://orcid.org/0000-0002-8824-4637",
                    "affiliation": "George Washington University",
                    "contribution": [
                        "curatedBy"
                    ],
                    "name": "Janisha Patel",
                    "email": "janishapatel@gwu.edu"
                },
                "date": "10/16/2019"

### Once all these fields are empty in your BCO editor, download and save. Then you can submit to the pFDA challenge.